lib/page/assets/index.dart

```diff
@@ -500,7 +500,11 @@ class _AssetsState extends State<Assets> {
                         : null,
                   ),
                 )
-              : Container();
+              : RoundedCard(
+              margin: EdgeInsets.only(top: 16),
+              padding: EdgeInsets.symmetric(vertical: 8),
+              child: ListTile(title: CupertinoActivityIndicator())
+          );
```

Comment:
I think this is actually misleading. This case can occur when there has not been a community selected yet. If an activity indicator is shown, the user might want to wait.

However, it makes sense to change that. I would display a text here e.g.:

```suggestion
 RoundedCard(
              margin: EdgeInsets.only(top: 16),
              padding: EdgeInsets.symmetric(vertical: 8),
              child: chosenCid == null ? Text("No community chosen") : CupertinoActivityIndicator()
);
```

Why did you introduce here the `ListTile`?

Reply:
No community chose is wrong if one has been chosen but we just wait for the update. In that case it should be an activityindicator

Reply:
Yes, I badly articulated that. The suggested code change does:

* Display 'No community chosen' if `chosenCid == null`
* otherwise display `CupertinoActivityIndicator`

Reply:
well, I disagree on some points here. 
The way the code is written, a community will always be chosen, if there is one (first in the list), so the only time no community is shown is, if no communities are registered. Therefore I'd propose to display "no communities registered on chain {chainName}"
And then again, this would also account for the CommunityChooserPanel to display that no community is registered, rather than no community chosen

Reply:
For now I will display "no community chosen", but I also won't set the first community as the default value, like this, the user gets, that he needs to choose a community to display the ammount of tokens

Reply:
I am quite confident, it is not as you say, but we can check that together tomorrow.